users/templates/users/volunteeropp.html

```diff
@@ -0,0 +1,714 @@
+<!doctype html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta charset="UTF-8">
+	<link rel="stylesheet" type="text/css" href="../template.css">
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+
+
+	<title> Volunteer Opportunities </title>
+
+	<style>
```

Comment:
Does it make sense to pull styles to a separate CSS file? And maybe reconcile it with the CSS file @lisanag created?

(Not urgent, though, can be done as a separate PR.)